ext/node/ops/buffer.rs

```diff
@@ -0,0 +1,28 @@
+// Copyright 2018-2024 the Deno authors. All rights reserved. MIT license.
+
+use deno_core::op2;
+use deno_core::v8;
+
+#[op2(fast)]
+pub fn op_is_ascii(buf: v8::Local<v8::ArrayBufferView>) -> bool {
```

Comment:
Maybe `#[arraybuffer] &[u8]` + https://doc.rust-lang.org/std/primitive.slice.html#method.is_ascii ?

Reply:
Ah, good point. I forgot we even have helpers for array buffers.

For reference: https://github.com/denoland/deno_core/tree/main/ops/op2#parameters